src/sentry/static/sentry/app/views/organizationIntegrations/integrationUtil.tsx

```diff
@@ -0,0 +1,37 @@
+const pluginMapping = {
+  id: 'id',
+  name: 'name',
+};
+
+const providerMapping = {
+  id: 'key',
+  name: 'name',
+};
+
+const sentryAppMapping = {
+  id: 'slug',
+  name: 'name',
+};
+
+const getMapping = type => {
```

Comment:
@NisanthanNanthakumar Would explicitly set the type of `type` instead of using the implicit `any`.